src/typescript/frontend/src/utils/formatter.ts

```diff
@@ -76,3 +76,12 @@ export const averageOrOtherPriceLevel = (
   // no prices (orderbook empty) maybe should get the last sale price then?
   return { price: 0, size: 0 };
 };
+
+export const canBeBigInt = (num: string | number | bigint | boolean) => {
+  try {
+    BigInt(num);
+    return true;
+  } catch (e) {
+    return false;
+  }
+};
```

Comment:
Not really a fan of using try catch in this way - I think if you are going to use a try catch block you should just do it in the original function

But in this case, I think `Number.isInteger(Number(num))` does what you need?

Reply:
Sure, I can move it -- `Number.isInteger(Number(100000000000000000000000000.1))` evaluates to `true`

Reply:
oh lol